### PR TITLE
Upgrade Buildbarn components 2022-09-24

### PR DESCRIPTION
**Pull request by moroten**

### CODE REVIEW

File: tools/expected-docker-and-k8s-configs.diff [hunk 1]
```diff
@@ -53,21 +53,21 @@ index 0000000..7f08b6d 100644
...
 --- a/-
 +++ b/docker-compose/config/worker-ubuntu22-04.jsonnet
-@@ -6,9 +6,7 @@ local common = import 'common.libsonnet';
+@@ -5,9 +5,7 @@ local common = import 'common.libsonnet';
+   browserUrl: common.browserUrl,
    maximumMessageSizeBytes: common.maximumMessageSizeBytes,
    scheduler: { address: 'scheduler:8983' },
-   maximumMemoryCachedDirectories: 1000,
 -  global: common.global + {
 -    setUmask: { umask: 0 },
```
Comment (EdSchouten) [inline tools/expected-docker-and-k8s-configs.diff:64]: Are you certain that this is no longer needed?

**Review by EdSchouten — APPROVED** [review]

**Comment by EdSchouten** [comment]
Thanks for working on this!

File: tools/expected-docker-and-k8s-configs.diff (hunk 1 above)
Comment (moroten) [reply tools/expected-docker-and-k8s-configs.diff:64]: It's a change of a diff file. `maximumMemoryCachedDirectories` was removed from the context and `browserUrl` came in instead.

Comment (EdSchouten) [reply tools/expected-docker-and-k8s-configs.diff:64]: Oh merp. Gotcha! Looks good!

**Pull request merged**